examples/double.html

```diff
@@ -25,31 +25,41 @@
             <div class="col-md-3">
                 <div class="form-group">
                     <div class="input-group" datetimepicker
-                         on-change="vm.optionsTo.minDate = vm.dateFrom"
+                         on-change="vm.onChangeDatePickerFrom()"
```

Comment:
Please keep the old tests untouched.

1.  All new features are optional, so I believe the old scenarios should work?
2. Please provide NEW tests for new features